source/api/commands/task.md

```diff
@@ -0,0 +1,177 @@
+---
+title: task
+comments: false
+---
+
+Execute code in node.js via the `task:requested` plugin event.
+
+{% note warning 'Anti-Pattern' %}
+Don't try to start a web server from `cy.task()`.
+
+Read about {% url 'best practices' best-practices#Web-Servers %} here.
+{% endnote %}
+
+# Syntax
+
+```javascript
+cy.task(event)
+cy.task(event, arg)
+cy.task(event, arg, options)
+```
+
+## Usage
+
+**{% fa fa-check-circle green %} Correct Usage**
+
+```javascript
+// in test
+cy.task('log', 'This will be output to the terminal')
```

Comment:
Oh boy, I worry that this will confuse people between this and `cy.log()` and like, console.log to console....I worry that this example may not be evident enough that this is a log in Node. Is there another simple example we could use that doesn't involve `console.log` that communicates, this is NODE?

Reply:
Yeah, it's hard to come up with terse, non-convoluated examples for this. I like this example because it's something users have asked for (to log to the terminal) and now they'll be able to do it pretty easily. Plus, maybe it's good for them to ask how this is different from `cy.log`? It's already confusing that `cy.log` logs to one place and not to another...

Reply:
Uh then let's just make `cy.log` log out to `stdout` like... right now, lol...